calculator.js

```diff
@@ -6,21 +18,75 @@ const exampleAdditionInput = {
 }
 
 const calculateUserInput = function (error, promptInput) {
-  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
-
-  // Questions to ask and answer:
-  // What is promptInput?
-  // What data type? What does it hold? What does it represent?
-  // How do we read values from it? What syntax?
-  // How can we use it?
-  // Can we call our existing functions now, inside of this function?
-}
+  console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput,'\n');
+  
+  const firstNum = parseFloat(promptInput.num1);
+  const secondNum = parseFloat(promptInput.num2);
+  const operator = promptInput.operation.toLowerCase();
+  let result = 0;
+
+  if (isNaN(promptInput.num1) || isNaN(firstNum)) {
+    return console.log(`${promptInput.num1} is not a number.`);
+  }
+  else if (isNaN(promptInput.num2) || isNaN(secondNum)) {
+    return console.log(`${promptInput.num2} is not a number.`);
+  };
+
+  switch (operator) {
+    case 'add': case '+':
+      result = firstNum + secondNum;
```

Comment:
Hmm, do we *need* a `result` variables? Can we execute these operations without declaring a `result` variable?